packages/model-registry/upstream/frontend/src/app/pages/modelRegistry/screens/RegisteredModels/RegisteredModelTableRow.tsx

```diff
@@ -61,7 +67,22 @@ const RegisteredModelTableRow: React.FC<RegisteredModelTableRowProps> = ({
         );
       },
     },
+    {
+      title: 'Deployments',
+      onClick: () => {
+        navigate(`${rmUrl}/deployments`);
+      },
+    },
+
+    { isSeparator: true },
+
+    {
+      title: 'Latest version actions',
+      isDisabled: true,
+    },
```

Comment:
Hmm, a disabled action as a group header isn't ideal since PF now supports [DropdownGroups](https://www.patternfly.org/components/menus/dropdown#with-groups-of-items), but I guess we're stuck with these IAction objects for now and refactoring that is out of scope here. Something else to come back to some day.

Reply:
Hmm - it looks like the archive and restore actions that also exist upstream should still be labeled "Latest version actions" now, right? so maybe this separator/title should be added upstream instead? Not a blocker, but something we might want to port over at some point.

Reply:
Deploy and LAB-tune are the "latest version" actions and these only exist midstream 😅